docs/FAQ.md

```diff
@@ -93,7 +93,7 @@ Alternatively, you can use [jj-fzf](https://github.com/tim-janik/jj-fzf), where
 
 The wiki lists additional TUIs and GUIs beyond the terminal: [GUI-and-TUI](https://github.com/jj-vcs/jj/wiki/GUI-and-TUI)
 
-### <a name="should-i-co-locate-my-repository"></a>Should I colocate my repository?
+### <a name="should-i-colocate-my-repository"></a>Should I colocate my repository?
```

Comment:
These are not links. These make it so that this heading could be reached by either the old name or the new name.

Reply:
In other words, this is our attempt to *avoid* hyperlinks breaking.

Reply:
That is exactly what I meant by my PR comment. I noticed that #7419 broke some links to the `colocated-jujutsugit-repos` anchor in git-compatibility.md so this PR fixes that. At the same time I thought it'd make sense to also update the few remaining places where the anchors still used co-locate instead of colocate, since (I think that) every other anchor was changed by #7419.

Reply:
I'm not sure that I follow. Both of these links work for me:

https://jj-vcs.github.io/jj/prerelease/git-compatibility/#colocated-jujutsugit-repos
https://jj-vcs.github.io/jj/prerelease/git-compatibility/#co-located-jujutsugit-repos

The HTML is:

```html
<h2 id="colocated-jujutsugit-repos">
  <a name="co-located-jujutsugit-repos"></a>
  Colocated Jujutsu/Git repos
  <a class="headerlink" href="https://jj-vcs.github.io/jj/prerelease/git-compatibility/#colocated-jujutsugit-repos" title="Permanent link">¶</a>
</h2>
```

One thing I noticed about #7419 that it didn't add `<a name` everywhere it changed the names of the headings, though.

Reply:
Are the links broken in the GitHub UI, perhaps?

Reply:
> One thing I noticed about #7419 that it didn't add `<a name` everywhere it changed the names of the headings, though.

I only added the new anchors for those links which were referenced by other documents in the repository. I thought it might be very cumbersome to maintain links for all changed heading names. (I can’t imagine we want to do that for every single heading in the future, but I thought it could be useful for direct links referenced by the docs.)

Reply:
> Are the links broken in the GitHub UI, perhaps?

I could not get the links to `colocated-repos` on concurrency.md to work when I clicked on them from my editor unless I changed the corresponding anchor from `co-located-repos` to `colocated-repos`. I am surprised that they work on the documentation site but they do.

Given that's the case I guess we can just close this PR since it does not seem to be needed...

Reply:
Which editor? I wonder whether this is a bug in one specific editor or some intentional behavior I don't understand.

Did changing the anchor make a difference? Do other links work in that editor?

I think we should probably close this, yes. If we can figure out some way we can improve things, we can do it separately.

Reply:
VS Code. Other links worked fine and I had to change the colocated-repo links to make them work, but perhaps there was something else causing the issue. I'm going to close the PR. Thank you for having a look!